.github/workflows/deploy.yml

```diff
@@ -6,45 +6,15 @@ on:
   push:
     branches:
       - main
-jobs:
-  Dialyzer:
-    strategy:
-      matrix:
-        elixir: ['1.13']
-        otp: [25]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: set cache key
-        id: cache_key
-        run: |
-          mix_hash="${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}"
-          echo "::set-output name=mix_hash::$mix_hash"
-      - uses: actions/cache@v2
-        with:
-          path: |
-            _build/dev/*.plt
-            _build/dev/*.plt.hash
-          key: plt-cache-${{ steps.cache_key.outputs.mix_hash }}
-          restore-keys: |
-            plt-cache-
-      - uses: erlef/setup-beam@v1
-        with:
-          otp-version: ${{matrix.otp}}
-          elixir-version: ${{matrix.elixir}}
-      - run: mix deps.get
-      - run: mix dialyzer
 
+jobs:
   Deploy:
     if: ${{ github.repository_owner == 'elixirschool' }}
     runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v3
-
-      - name: Setup Fly
-        uses: superfly/flyctl-actions/setup-flyctl@master
-
-      - name: Deploy
-        run: fly deploy --remote-only
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+      - run: flyctl deploy --remote-only
```

Comment:
This should fix the deploy command. I'm assuming they changed some alias from `fly` -> `flyctl` so it no longer exists :cry: